.gitignore

```
@@ -0,0 +1,4 @@
+.env
+portal/
```

Add the explorer too
```suggestion
portal/
explorer/
```